allure-framework-modules/allure-selenium-module/src/main/java/com/capgemini/ntc/selenium/core/newDrivers/elementType/ListElements.java

```diff
@@ -28,4 +29,11 @@ public Integer getSize() {
 				.findElementDynamics(this.cssSelector);
 	}
 	
+	public List<String> getTextList() {
+		List<String> list = new ArrayList<String>();
```

Comment:
You can try to use stream().map() here-> https://www.mkyong.com/java8/java-8-streams-map-examples/